VMEncryption/main/EncryptionSettingsUtil.py

```diff
@@ -182,14 +182,39 @@ def get_http_util(self):
         from HttpUtil import HttpUtil
         return HttpUtil(self.logger)
 
+    def get_wireserver_endpoint_uri(self):
+
+        wireserver_endpoint_file = CommonVariables.wireserver_endpoint_file
+        wireserver_IP = None
+        wireserver_endpoint_uri = CommonVariables.wireserver_endpoint_uri
+
+        if os.path.exists(wireserver_endpoint_file):
+            with open(wireserver_endpoint_file, 'r') as wip:
+                wireserver_IP = wip.readline().strip()
+                self.logger.log("wireserver_IP found in {0} = {1}".format(wireserver_endpoint_file, wireserver_IP))
+
+                # validate the IP address found in wireserver_endpoint_file
+                if re.match('\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}', wireserver_IP) is None:
+                    wireserver_IP = None
+                    self.logger.log("wireserver_IP found in file is not valid.")
+
+        if wireserver_IP is None:
+            self.logger.log("Using static wireServer_IP from CommonVariables")
+            wireserver_IP = CommonVariables.static_wireserver_IP
+
+        wireserver_endpoint = "http://" + wireserver_IP + wireserver_endpoint_uri
+        self.logger.log("wireserver_endpoint = {0}".format(wireserver_endpoint))
+        return wireserver_endpoint
+
     def _post_to_wireserver_helper(self, msg_data, http_util):
 
         retry_count_max = 3
         retry_count = 0
+        wireserver_endpoint_uri = self.get_wireserver_endpoint_uri()
         while retry_count < retry_count_max:
             try:
                 result = http_util.Call(method='POST',
-                                        http_uri=CommonVariables.wireserver_endpoint,
+                                        http_uri=wireserver_endpoint_uri,
                                         headers=CommonVariables.wireprotocol_msg_headers,
                                         data=msg_data,
                                         use_https=False)
```

Comment:
In case of failure, consider including a status message that includes an aka.ms redirect that goes to status documentation such as the following to help with network configuration: 

https://docs.microsoft.com/en-us/azure/virtual-machines/windows/disk-encryption-troubleshooting#troubleshooting-azure-disk-encryption-behind-a-firewall

https://docs.microsoft.com/en-us/azure/virtual-network/what-is-ip-address-168-63-129-16

Reply:
Thats a good point. I will add this as part of another PR that will standardize error messages for migrate operation.